---
layout: default
---


# About

I recently completed my PhD in Bioinformatics at Temple University, advised by [Sergei Pond](https://igem.temple.edu/people/person/e266d9a5b7f043109baecc3c340491f6) in the [Viral Evolution Group.](http://lab.hyphy.org/) A part of the Institute for Genomics and Evolutionary Medicine [iGEM](https://igem.temple.edu/) 

My research largely encompasses the development and validation of statistical methods and understanding how natural selection shapes gene evolution.This includes understanding complex multi-nucleotide mutation changes within molecular sequences. I also have an interest in computational virology, including the development of workflows and applications that enable the near-real-time monitoring of the population-level selective processes that underlie the emergence of viral variants in rapidly evolving pathogens with extensive genomic surveillance. 

Previously, I earned a B.S. in Biochemistry from Stony Brook University and M.S. in Biology at New York University.

I completed my undergraduate studies at Stony Brook University, where I earned a Bachelor of Science (B.S.) degree in Biochemistry and obtained a Master of Science (M.S.) degree in Biology from New York University.

I committed to mentoring and inspiring the next generation of scientists by sharing knowledge and expertise with aspiring researchers and fostering their passion for scientific inquiry. Myself and several other graduate students cofounded the [Bioinformatics Studio](https://tubioinfostudio.github.io), an inclusive and hands-on environment for learning, training, sharing, and most importantly doing bioinformatics.

Feel free to contact me via email at alexander.lucaci at temple dot edu. You can also connect with me on [Twitter.](https://twitter.com/aglucaci)

# Curriculum Vita
*   [Link to my current CV](Resume/CV_AGL_2023.pdf)

# Publications
An up-to-date list can be found on [my Google scholar page.](https://scholar.google.com/citations?user=Bnxc_OoAAAAJ)

***Preprints***

* Evolutionary shortcuts via multi-nucleotide substitutions and their impact on natural selection analyses <br />
Alexander G Lucaci, Jordan D Zehr,  David Enard, Joseph W. Thornton, Sergei L. Kosakovsky Pond <br />
doi: https://doi.org/10.1101/2022.12.02.518889

***Selected papers***
*  [Emergence of SARS-CoV-2 Omicron lineages BA.4 and BA.5 in South Africa](https://www.nature.com/articles/s41591-022-01911-2)
*  [Selection analysis identifies unusual clustered mutational changes in Omicron lineage BA.1 that likely impact Spike function](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msac061/6553617)
*  [Rapid epidemic expansion of the SARS-CoV-2 Omicron variant in southern Africa](https://www.nature.com/articles/s41586-022-04411-y)
*  [The emergence and ongoing convergent evolution of the N501Y lineages coincides with a major global shift in the SARS-CoV-2 selective landscape](https://www.cell.com/cell/fulltext/S0092-8674(21)01050-3)
*  [RASCL: Rapid Assessment of SARS-CoV-2 CLades through molecular sequence analysis](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0275623)
*  [Evolutionary shortcuts via multi-nucleotide substitutions and their impact on natural selection analyses.](https://www.biorxiv.org/content/10.1101/2022.12.02.518889v1)
*  [The phylogenomics and evolutionary dynamics of the organellar genomes in carnivorous Utricularia and Genlisea species (Lentibulariaceae)](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4239133)
*  [The Evolution of Brain-Derived Neurotrophic Factor (BDNF) Reveals Insight into Brain Disorder Vulnerability](https://www.nature.com/articles/s41398-022-02021-w)
*  [Human HspB1, HspB3, HspB5 and HspB8: Shaping these Disease Factors during Vertebrate Evolution](https://link.springer.com/article/10.1007/s12192-022-01268-y)
*  [Extra base hits: Widespread empirical support for instantaneous multiple-nucleotide changes](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248337)
*  [The evolution of BDNF is defined by strict purifying selection and prodomain spatial coevolution, but what does it mean for human brain disease?](https://www.nature.com/articles/s41398-022-02021-w)
*  [Electrical synaptic transmission in developing zebrafish: properties and molecular composition of gap junctions at a central auditory synapse](https://journals.physiology.org/doi/full/10.1152/jn.00397.2014)
*    Acknowledgement: [Human Embryonic Stem Cell-Derived Oligodendrocyte Progenitors Remyelinate the Brain and Rescue Behavioral Deficits following Radiation](https://www.cell.com/cell-stem-cell/fulltext/S1934-5909(15)00005-3)

**Additional research in SARS-CoV-2**
*  [Selection analysis identifies significant mutational changes in Omicron that are likely to influence both antibody neutralization and Spike function (Part 1 of 2)](https://virological.org/t/selection-analysis-identifies-significant-mutational-changes-in-omicron-that-are-likely-to-influence-both-antibody-neutralization-and-spike-function-part-1-of-2/771)
*  [Selection analysis identifies significant mutational changes in Omicron that are likely to influence both antibody neutralization and Spike function (Part 2 of 2)](https://virological.org/t/selection-analysis-identifies-significant-mutational-changes-in-omicron-that-are-likely-to-influence-both-antibody-neutralization-and-spike-function-part-2-of-2/772)
*  [Assessing evolutionary pressures on the SARS-CoV-2 Î¼ (B.1.621) clade](https://virological.org/t/assessing-evolutionary-pressures-on-the-sars-cov-2-mu-b-1-621-clade/760)
*  [Rapid molecular sequence characterization of the SARS-CoV-2 variant of interest Lambda (C.37)](https://virological.org/t/rapid-molecular-sequence-characterization-of-the-sars-cov-2-variant-of-interest-lambda-c-37/752)
*  [RASCL: Rapid assessment of SARS-COV-2 clades enabled through molecular sequence analysis and its application to B.1.617.1 and B.1.617.2](https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709)
*  [RASCL](https://observablehq.com/@aglucaci/rascl_latest)
*  [RASCL Heavy](https://observablehq.com/@aglucaci/rascl_heavy_latest)

# Teaching

**Science education and outreach**
* Co-Founder, [Bioinformatics Studio](https://tubioinfostudio.github.io/)
* Graduate Teaching Assistant, Genomics in Medicine 

**Science communication**
* [Galaxy Community Conference (GCC) 2022](https://docs.google.com/presentation/d/1j94262RzoNoH3s3NfMjVddfUG53GhIAz/present?rtpof=true&sd=true&slide=id.g13b6e32f2ef_15_0)
* [29th International Dynamics & Evolution of Human Viruses](https://docs.google.com/presentation/d/14NAL1mMucNFgwTYKjZvaAN4TzZ75qKN4I93jI7UXR3Y/present#slide=id.p)
* [2019 MidAtlantic Bioinformatics Conference](https://aglucaci.github.io/Resume/MH_Poster.pptx.pdf)
* [28th International Dynamics & Evolution of Human Viruses](https://docs.google.com/presentation/d/18Nd-Lhy-vwYlWO9F-jEl4stRMlTVjYjT7NBPghigS6I/present?slide=id.p)
* [5th Workshop on Virus Dynamics](https://aglucaci.github.io/Resume/AGL_5thWorkshopOnVirusDynamics.pdf)
* 2021 Society for Molecular Biology & Evolution
* [2021 MidAtlantic Bioinformatics Conference](https://aglucaci.github.io/Resume/AGL_5thWorkshopOnVirusDynamics.pdf)
* 2021 EMBO Workshop: The Evolution of Animal Genomes

**Press coverage**
* [Selective pressure is contributing to faster development of Covid-19 variants](https://cst.temple.edu/news/2021/12/selective-pressure-contributing-faster-development-covid-19-variants)

**Presentation Slides**
*   [Research overview 2022](https://tuprd-my.sharepoint.com/:p:/g/personal/tuk13147_temple_edu/ESOcJ8bz7HVFkZnvU2H3ZcgBr736j3Ei8kYqroSp3m_QTQ?e=VeUzQF)
*   [Research overview 2022 (GoogleDrive Backup)](https://docs.google.com/presentation/d/1G7CMn6ZJJxgzCEuyWnb3SurGU1uWLAL_/present?rtpof=true&sd=true&slide=id.p1)
*   [Dissertation Defense Slides (2023)](https://docs.google.com/presentation/d/14voBHpB4F9C913gqpIZzd1V0DzAg8keX/present?rtpof=true&sd=true&slide=id.p1)
*   [30th International Dynamics & Evolution of Human Viruses (2023)](https://docs.google.com/presentation/d/16SLKLta0QywoCygagqAOIqPkV2BWgLc11gcgeqr8OLQ/present?usp=sharing)

**Masters Thesis**
*   [The twilight of tobacco: how electronic cigarettes can change humanity's relationship with nicotine](https://drive.google.com/file/d/1t12NHLDER5Qxk8LCIBVqvUzTRKg-l8CM/view?usp=sharing)

# Software
*   [Multi-hit codon models](https://www.datamonkey.org/multihit)
*   [Multiple sequence alignment profiler](https://colab.research.google.com/drive/1kOemj6hyH4tKmo_fcztoG95ipv3iovxP?usp=sharing)
*   [Analysis of Orthologous Collections (AOC)](https://github.com/aglucaci/AnalysisOfOrthologousCollections)
*   [Molecular Evolution SubSampling Investigator (MESSI)](https://github.com/aglucaci/SubsamplingSequences)
*   [Measuring gene stoichiometry](https://github.com/veg/hyphy-examples/tree/master/gene_stoichiometry)
*   Rapid Assessment of SARS-CoV-2 CLades through molecular sequence analysis [RASCL](https://github.com/veg/RASCL)
*   [BUSTED Modeltesting](https://github.com/veg/BUSTED_ModelTest)





